galaxykit/groups.py

```diff
@@ -72,6 +73,7 @@ def delete_permission(client, group_name, permission):
     group_id = get_group(client, group_name)["id"]
     permissions_url = f"_ui/v1/groups/{group_id}/model-permissions/"
     resp = client.get(permissions_url)
+    pprint(resp)
```

Comment:
so.. was this intentional?

(looks like you removed the one pprint after last review, but then added it here?)